dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/EventDataQueryRequest.java

```diff
@@ -66,9 +66,9 @@
 
     private Date endDate;
 
-    private Set<String> dimension;
+    private Set<Set<String>> dimension;
```

Comment:
Does this support things like `programStageUid.ElementUid`, or only single `uid`? Is it also been taken into consideration during query time?

Reply:
Yes everything is as it was before, the only difference is that we now split any dimension param on _OR_ and assign the same group to it